WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsSectionHeaderView.xib

```diff
@@ -53,7 +54,7 @@
     </objects>
     <resources>
         <namedColor name="Gray50">
-            <color red="0.40392156862745099" green="0.41568627450980394" blue="0.45490196078431372" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+            <color red="0.39215686274509803" green="0.41176470588235292" blue="0.4392156862745098" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
```

Comment:
I'm not sure why Xcode changes these slightly, but I don't think it's an issue because Gray50 is still rgb(100, 105, 112) both in this branch and develop. 🤷‍♂️